src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/perform_interaction_request.cc

```diff
@@ -92,7 +93,9 @@ bool PerformInteractionRequest::Init() {
 
   if (mobile_apis::InteractionMode::BOTH == interaction_mode_ ||
       mobile_apis::InteractionMode::MANUAL_ONLY == interaction_mode_) {
-    default_timeout_ *= 2;
+    default_timeout_ = default_timeout_ + (request_timeout_ * 2);
```

Comment:
@ValeriiMalkov use +=

Reply:
@AKalinich-Luxoft Fixed in commit 76a1afadfa7423d3c12719ea6f4e49a302c04395